README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
```

Comment:
Evaluation HW-01-Amanda Brown
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |         X  |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |           |        X      |            |
| **Ease of access**          |   X        |              |            |

Remarks:
Amanda, this biography looks great. You used at least four of the elements that were listed on the homework and I like the way you organized your biography by sections using the varied coding for header. I think that's all I have to say about your biography. Regarding your comments about your process, I was also slightly confounded by the concept of commit and how it differs from save, and why frequent commitment is good practice. Good to know I'm not the only person thinking about this, and perhaps this could be brought up in class tomorrow.